build/gulp-config.json.dist

```diff
@@ -1,5 +1,5 @@
 {
-	"wwwDir"        : "/var/www/html/yoursite",
+	"wwwDir"        : "/home/travis/build/redCOMPONENT-COM/redCORE/tests/joomla-cms3",
```

Comment:
also this is just for watcher and browser sync, it does not have an impact on release task